tensorflow/compiler/mlir/xla/transforms/hlo_legalize_to_lhlo.cc

```diff
@@ -316,10 +316,13 @@ class HloToLhloTensorStoreOpConverter : public ConversionPattern {
 //               %arg1: memref<4xf32>,
 //               %arg2: memref<4xf32>) {
 //   %0 = alloc() {temp = true} : memref<4xf32>
+//   %1 = alloc() {temp = true} : memref<4xf32>
```

Comment:
if `temp` attribute is not used, please, remove it

Reply:
Done.